### ISSUE TITLE
Unable to prisma deploy or yarn dev (prisma/playground) after 1.7.1 upgrade. Running on Prisma Cloud

### ISSUE THREAD

**Issue by k00k** [graphcool/prisma-cloud-feedback#137]
I have updated `prisma` to `1.7.1` (local and global)
and updated  `graphql-cli` to `2.15.11` (local and global)

After upgrading, `prisma deploy` prompted me to edit my `prisma.yml` file to remove the old settings (cluster, service, etc) and add `endpoint`

My `prisma.yml` now only contains:

```
endpoint: ${env:PRISMA_ENDPOINT}
secret: ${env:PRISMA_SECRET}
datamodel: datamodel.graphql
```

`prisma deploy` now yields:

```
▸    Your `cluster` property in the prisma.yml is missing the workspace slug.
▸    Make sure that your `cluster` property looks like this: <workspace>/<cluster-name>. You can also remove the cluster property from the prisma.yml
▸    and execute prisma deploy again, to get that value auto-filled.
```

`yarn dev` yields:
```
$ npm-run-all --parallel local-start playground
yarn run v0.21.3
yarn run v0.21.3
$ graphql playground --port 5000
$ dotenv -- nodemon -e ts,graphql -x ts-node src/index.ts
[nodemon] 1.17.3
[nodemon] to restart at any time, enter `rs`
[nodemon] watching: *.*
[nodemon] starting `ts-node src/index.ts`
(node:2192) UnhandledPromiseRejectionWarning: Error: Your `cluster` property in the prisma.yml is missing the workspace slug.
Make sure that your `cluster` property looks like this: <workspace>/<cluster-name>. You can also remove the cluster property from the prisma.yml
and execute prisma deploy again, to get that value auto-filled.
    at PrismaDefinitionClass.validate (/private/var/dev/v4/api/node_modules/graphql-config-extension-prisma/node_modules/prisma-yml/src/PrismaDefinition.ts:189:13)
    at PrismaDefinitionClass.<anonymous> (/private/var/dev/v4/api/node_modules/graphql-config-extension-prisma/node_modules/prisma-yml/src/PrismaDefinition.ts:77:12)
    at step (/private/var/dev/v4/api/node_modules/graphql-config-extension-prisma/node_modules/prisma-yml/dist/PrismaDefinition.js:32:23)
    at Object.next (/private/var/dev/v4/api/node_modules/graphql-config-extension-prisma/node_modules/prisma-yml/dist/PrismaDefinition.js:13:53)
    at fulfilled (/private/var/dev/v4/api/node_modules/graphql-config-extension-prisma/node_modules/prisma-yml/dist/PrismaDefinition.js:4:58)
    at <anonymous>
```

I saw this commit by @timsuchanek which doesn't seem to have landed yet, but seems related: https://github.com/graphcool/prisma/commit/611dfcc22c67a0b9ab1150d0fd34e01b7d0760be

Please let me know if I can supply any additional info.

**Comment by timsuchanek** [comment]
Thanks for reporting @k00k! I just tried deploying to a prisma cloud cluster with 1.7.1 and can't reproduce your issue. Can you please share the content of $PRISMA_ENDPOINT with me? You can either obfuscate the service & stage name (I'm just interested in the structure of the endpoint) or send it to me in Slack @tim2. Thanks!

**Comment by timsuchanek** [comment]
We just released a fix for this: `prisma@1.7.2`
